### ISSUE TITLE
Define a platform for the collection

### ISSUE THREAD

**Issue by sonialindblom** [sonialindblom/dch#5]
Should we just write a website from scratch and store it here?

**Comment by s1cheng** [comment]
How many photos do we have? I only find two photos here.

**Comment by s1cheng** [comment]
We may find some good examples of how the website should be like, and we can start to write it.

**Comment by sonialindblom** [comment]
> How many photos do we have? I only find two photos here.

Hi. The photos were for inspiration. I fixed a few folders now. The preview folder is with the pictures we are going to use, I used a flatbed scanner. 

It's 11 cards (two of them are not real cards, but I will probably find them)  so far and 1 letter from Swedish army.

**Comment by sonialindblom** [comment]
I have been thinking of getting one of google clouds-account and run Wordpress.
Of course a simple code will do just fine here in git-hub.

**Comment by s1cheng** [comment]
Yes, of course. Here is my google account:  s206110@student.hb.se
Btw, did you use any materials from the final project for the OCR or Tei tasks? I was thinking to use these materials so that we can save time for the final project.  But I did not find items that suit the tasks so well.

**Comment by sonialindblom** [comment]
For the TEI assignment I don't think we can use much, since we won't have
1000 words. It's mostly a letter and the texts in the postcards, that are
just a few words.
*/sonialindblom*
